spec/unit/environments_spec.rb

```diff
@@ -401,11 +402,12 @@ def remove
         base_dir = File.expand_path("envdir")
         original_envdir = FS::MemoryFile.a_directory(base_dir, [
           FS::MemoryFile.a_directory("env3", [
-            FS::MemoryFile.a_regular_file_containing("environment.conf", <<-EOF)
+            FS::MemoryFile.a_regular_file_containing("environment.conf", <<-EOF),
               manifest=/manifest_orig
-              modulepath=/modules_orig
```

Comment:
Why does this slash go a way, but we keep the one for `manifest_orig`?

Reply:
Yeah that's a good question. The `Environment#modules` method now uses `Puppet::FileSystem.children` instead of `Dir.entries` to list the modules in the environment. But the memory Puppet::FileSystem implementation used in tests will raise if the a file/directory is accessed, but hasn't been registered via `a_directory`, etc. So I modified `modules_orig` to be relative within the environment directory `env3` and then registered it below in https://github.com/puppetlabs/puppet/pull/8673/files#diff-e665c46dc177d4c7e13d6ff5f107c10926a471b9913b1bdd9bdfea63a31e0148R410

Reply:
Oh I see, but `manifest_orig` isn't accessed as part of that, so it's fine like it was. Thanks.